Source/Internal/Handle/DrivingScrollViewViewModifier.swift

```diff
@@ -14,7 +14,7 @@ struct DrivingScrollViewViewModifier: ViewModifier {
     let isActive: Bool
 
     func body(content: Content) -> some View {
-        DrivingScrollViewWrapper(content: content)
+        DrivingScrollViewWrapper(content: ZStack { content })
```

Comment:
Why do you need a `ZStack` here?

Reply:
I was trying to prevent a situation where the user was setting an accessibility identifier in SwiftUI and we would override that identifier by setting the accessibilityIdentifier. After testing this it appears that SwiftUIs accessibilityIdentifier does in fact not set the accessibilityIdentifier of the UIKit-View. So I removed the ZStack.